### PR TITLE
added blank ci

### LINKED ISSUE #3: CI Tool : Github Action
The requirements could be like below
 1 - Easy to use
 2 - Fully Github Integration (Needs to block the Pull Requests which do not pass all CI tests)
 3 - Free

Jenkins is widely used for CI but it is awful so we can use Travis-CI here which is fully integrated with Github and free for Open-Source Projects and widely used by open source projects too.

### PR DESCRIPTION
**Pull request by alifiratari**
CI: added blank yml file

### CODE REVIEW

**Comment by alifiratari** [comment]
image

File: .github/workflows/blank.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# This is a basic workflow to help you get started with Actions
+name: CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [ T-8 ]
+  pull_request:
+    branches: [ T-8 ]
```
Comment (zaya-mc) [inline .github/workflows/blank.yml:10]: @alifiratari Should they be "master" before we merge to master branch?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,4 @@
 # EmbeddedTemplate
 Template Project For Embedded Projects
+
+![CI](https://github.com/alifiratari/EmbeddedTemplate/workflows/CI/badge.svg?branch=T-8)
```
Comment (zaya-mc) [inline README.md:4]: What it is for?  If it is required can you please add comments?

**Review by zaya-mc — COMMENTED** [review]
@alifiratari Please do not fork the repo, instead please create a branch with the ticket number; T-3 in your case. (the GitHub Ticket ; "Discussion : Continuous Integration Tool #3")

Maybe it is just a test and not a proper Pull Request. If it is a proper pull request, please have a look at Development Guide ; https://github.com/OpenIoTAcademy/EmbeddedTemplate/blob/T-2/Documents/GitTutorial.md

Please see Commit Comment Convention section. 

Thanks.

**Comment by zaya-mc** [comment]
@alifiratari  I have just merged T-8 branch to master, can you please rebase from master and update your CI script if it was dependent to T-8. Thanks.

**Pull request merged**